pages/common/sync.md

```diff
@@ -0,0 +1,11 @@
+# sync
+
+> Synchronises all pending write operations to the appropriate disk.
```

Comment:
Maybe its better to use the word "flush". I know "synchronises" matches with the command, but filesystem and the buffer cache are not meant to be in sync at all. It all totally depends on the OS implementation.

Reply:
Yeah, I agree flush makes it clearer.